button.js

```diff
@@ -119,14 +158,24 @@ async function moveToBugzilla(target) {
   const product = productDropdown.value;
 
   if (!component || !product) {
-    componentDropdown.style.display = 'initial';
-    productDropdown.style.display = 'initial';
+    for (dropdown of allDropdowns) {
```

Comment:
why the style change? And why in this case we check for product and component?

Reply:
I don't think there's actually a style change, it's just that the style change is applied to each dropdown via a loop rather than serially.